mailo/src/test/scala/mailo/PersistenceSpec.scala

```diff
@@ -94,15 +94,23 @@ class MockedData(implicit executionContext: ExecutionContext) extends MailData {
     Future(Right(MailRawContent("ciao", partials = Map("name" -> "claudio"))))
 }
 
+object SkipOnScala212 extends munit.Tag
+
 class PersistenceSpec extends {
   val system: ActorSystem = ActorSystem("testSystem")
 } with munit.FunSuite with TestKitBase with ImplicitSender {
 
-  override def afterAll: Unit = {
+  override def munitTestTransforms: List[TestTransform] =
+    super.munitTestTransforms :+ new TestTransform(
+      "skip on Scala 2.12",
+      test => if (test.tags.contains(SkipOnScala212)) test.withTags(munit.Tag.Ignore) else test,
+    )
```

Comment:
Small transform (🧙‍♂️) for skipping a specific test on 2.12. I've confirmed with @calippo that we don't really care about that test in general, but since it works reliably on 2.13 I've proceeded this way.